lib/std/c/freebsd.zig

```diff
@@ -862,6 +864,29 @@ comptime {
     assert(@sizeOf(kinfo_proc) == KINFO_PROC_SIZE);
 }
 
+pub const kinfo_vmobject = extern struct {
+    kvo_structsize: c_int,
```

Comment:
```suggestion
    structsize: c_int,
```
Remove the prefixes.